src/clients/java/src/test/java/com/tigerbeetle/AccountTest.java

```diff
@@ -147,6 +148,37 @@ public void testCodeOverflow() {
         accounts.setCode(Integer.MAX_VALUE);
     }
 
+    public void testReserved() {
```

Comment:
Why do some of these methods have `@Test` and others don't?

Reply:
Thank you for catching that ... I found a few others on the code, it's hard to spot!